lib/src/model/common/eval.dart

```diff
@@ -136,6 +142,26 @@ class PvData with _$PvData {
     }
     return res;
   }
+
+  MoveWithWinningChanges? firstMoveWithWinningChances(Side sideToMove) {
+    final uciMove = (moves.isNotEmpty) ? Move.fromUci(moves.first) : null;
+    return (uciMove != null)
+        ? MoveWithWinningChanges(
+            move: uciMove,
+            winningChances:
+                _toPov(sideToMove, _toWhiteWinningChanges(cp, mate)),
+          )
+        : null;
+  }
+}
+
+@freezed
+class MoveWithWinningChanges with _$MoveWithWinningChanges {
```

Comment:
Typo: should be `MoveWithWinningChances` with a C

Also I think here we could use a dart record instead of a new class:
```dart
typedef MoveWithWinningChances = (Move, double);
```

Reply:
Fixed the typo, sorry about that.

Regarding the record: IMO this makes the code less readable, since in `_computeBestMoveShapes` you'd either write something like `moves[index].$2` instead of `moves[index].winningChances`, or you'd need to use destructuring (`var (move, winningChanges) = moves[index]` (but then, why use a record in the first place instead of a class?)

Reply:
You can also defined a record with named fields: https://dart.dev/language/records#record-fields
```dart
typedef MoveWithWinningChances = ({ Move move, double winningChances });
```

Records are a language construct so I'd use them instead of `@freezed` classes whenever possible (when you need an immutable data structure without logic). Freezed classes bring overhead and more process time with `build_runner`.

Reply:
Ahh thanks for the clarification, didn't learn about that feature yet. Will change it accordingly